### PR TITLE
add "try it on ebrains" button for elephant tutorials

### PR DESCRIPTION
**Pull request by Moritz-Alexander-Kern**
This PR adds a ~~"Try it on EBRAINS"~~ launch ebrains next to "Launch on MyBinder" button for the elephant tutorials in the documentation.

Updated 30.01.2023
~~Currently the button used is from the NEST website:
https://nest-simulator.org/TryItOnEBRAINS.png~~

Example: (statistics tutorial)
[https://img.shields.io/badge/try%20it%20on-ebrains-brightgreen](https://lab.ch.ebrains.eu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2FNeuralEnsemble%2Felephant&urlpath=lab%2Ftree%2Felephant%2Fdoc%2Ftutorials%2Fstatistics.ipynb&branch=master)

Update 08.02.2023 changed text of shield:
[https://img.shields.io/badge/launch-ebrains-brightgreen](https://lab.ch.ebrains.eu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2FNeuralEnsemble%2Felephant&urlpath=lab%2Ftree%2Felephant%2Fdoc%2Ftutorials%2Fstatistics.ipynb&branch=master)

Documentation:
https://elephant--538.org.readthedocs.build/en/538/tutorials.html

### CODE REVIEW

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

File: doc/tutorials.rst [hunk 1]
```diff
@@ -4,8 +4,15 @@ Tutorials
 
 These tutorials provide narrative explanations, sample code, and expected
 output for some of the neurophysiological analyses in Elephant. You can browse
-the tutorials or launch them in mybinder to change and interact with the code.
+the tutorials, launch them in mybinder or try them on ebrains to change and
+interact with the code.
 
+Launching a notebook on ebrains and Binder both provide a convenient way to
```
Comment (mdenker) [inline doc/tutorials.rst:10]:
```suggestion
Launching a notebook on EBRAINS and Binder both provide a convenient way to
```

File: doc/tutorials.rst [hunk 2]
```diff
@@ -4,8 +4,15 @@ Tutorials
...
+run and interact with code. The main difference between the two is that
+changes made to a notebook launched on Binder are not saved, while
+changes made to a notebook launched on ebrains are persistent and bound to the
```
Comment (mdenker) [inline doc/tutorials.rst:13]:
```suggestion
changes made to a notebook launched on EBRAINS are persistent and bound to the
```

File: doc/tutorials.rst [hunk 3]
```diff
@@ -4,8 +4,15 @@ Tutorials
...
+user's ebrains account. This makes ebrains a great choice if you want to save
```
Comment (mdenker) [inline doc/tutorials.rst:14]:
```suggestion
user's EBRAINS account. This makes EBRAINS a great choice if you want to save
```

File: doc/tutorials.rst [hunk 4]
```diff
@@ -36,6 +44,8 @@ Advanced
 
   .. image:: https://mybinder.org/badge.svg
      :target: https://mybinder.org/v2/gh/NeuralEnsemble/elephant/master?filepath=doc/tutorials/unitary_event_analysis.ipynb
+  .. image:: https://img.shields.io/badge/launch-ebrains-brightgreen
```
Comment (mdenker) [inline doc/tutorials.rst:47]:
```suggestion
  ..
      .. image:: https://img.shields.io/badge/launch-ebrains-brightgreen
```

**Review by mdenker — CHANGES_REQUESTED** [review]

**Pull request merged**